app/javascript/mastodon/features/account/components/header.jsx

```diff
@@ -441,7 +441,7 @@ class Header extends ImmutablePureComponent {
             <h1>
               <span dangerouslySetInnerHTML={displayNameHtml} />
               <small>
-                <span>@{username}</span>
+                <span>@{username}<span class='invisible'>@{domain}</span></span>
```

Comment:
Oops, sorry about that:
```suggestion
                <span>@{username}<span className='invisible'>@{domain}</span></span>
```